app.js

```diff
@@ -16,8 +19,7 @@ const webOptions = {
   host: nconf.get('WEB_HOST')
 };
 
-const app = express();
-const server = http.createServer(app);
+app.use(express.static(path.join(__dirname, 'public/static')));
```

Comment:
public folder should suffice here:

app.use(express.static(path.join(__dirname, 'public')));